docs/tutorials/multimodal/object_detection/quick_start/quick_start_ovd.ipynb

```diff
@@ -0,0 +1,247 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "84a147df",
+   "metadata": {},
+   "source": [
+    "# AutoMM Detection - Quick Start with Foundation Model on Open Vocabulary Detection (OVD)\n",
+    "\n",
+    "[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/autogluon/autogluon/blob/master/docs/tutorials/multimodal/object_detection/quick_start/quick_start_ovd.ipynb)\n",
+    "[![Open In SageMaker Studio Lab](https://studiolab.sagemaker.aws/studiolab.svg)](https://studiolab.sagemaker.aws/import/github/autogluon/autogluon/blob/master/docs/tutorials/multimodal/object_detection/quick_start/quick_start_ovd.ipynb)\n",
+    "\n",
+    "\n",
+    "In this section, our goal is to use a foundation model in object detection to detect novel classes defined by an unbounded (open) vocabulary.\n",
+    "\n",
+    "## Setting up the imports\n",
+    "To start, let's import MultiModalPredictor:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "aa00faab-252f-44c9-b8f7-57131aa8251c",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "!pip install autogluon.multimodal\n",
+    "#!pip install git+https://github.com/IDEA-Research/GroundingDINO.git  # uncomment this line if this package is not installed yet"
```

Comment:
We can assume users already install automm? Instead, we should let users Install groundingdino?

Reply:
Done. Will try import grounding dino first, if not installed, will import pip and use pip.main